packages/run-protocol/test/amm/vpool-xyk-amm/setup.js

```diff
@@ -76,12 +76,12 @@ export const setupAMMBootstrap = async (
   produce.nameAdmins.resolve(nameAdmins);
 
   /** @type {Record<string, Promise<{moduleFormat: string}>>} */
-  const governanceBundlePs = {
+  const governanceBundlePs = harden({
```

Comment:
This is an example of a missing `harden` bug that's masked by the last version of `allValues`, where it auto-hardens its argument. The second version of `allValues` which is just an alias for `deeplyFulfilled` would fail immediately when applied to this unhardened record, quickly revealing rather than masking the bug.